api.c

```diff
@@ -160,6 +160,27 @@ void tcmu_seek_in_iovec(struct iovec *iovec, size_t count)
 	}
 }
 
+/*
+ * Consume an iovec. Count must not exceed the total iovec[] size.
+ * iove count should be updated.
+ */
+void tcmu_seek_in_cmd_iovec(struct tcmulib_cmd *origcmd, size_t count)
```

Comment:
origcmd should be renamed to just cmd, because it is a general function and my be not be used in the tcmur_cmd_handler specific way.